cli/src/template_builder.rs

```diff
@@ -436,6 +436,12 @@ fn build_timestamp_method<'a, L: TemplateLanguage<'a>>(
                 time_util::format_absolute_timestamp_with(&timestamp, &format)
             }))
         }
+        "UTC" => {
+            template_parser::expect_no_arguments(function)?;
+            language.wrap_string(TemplateFunction::new(self_property, |timestamp| {
+                time_util::format_to_utc(&timestamp)
+             }))
+        }
```

Comment:
As @yuja suggest on Discord, making `.utc()` return a new timestamp is more flexible. I think this should work:
```
        "utc" => {
            template_parser::expect_no_arguments(function)?;
            language.wrap_timestamp(TemplateFunction::new(self_property, |mut timestamp| {
                timestamp.tz_offset = 0;
                timestamp
            }))
        }
```